CREDITS.md

```diff
@@ -10,151 +10,152 @@
 
 ### TileSets
 
-- FantasyHex Merchant of Venice by Herobrine
 - Absolute Units by Basil
 - Brazilwood Camp by Kurwizimi
+- FantasyHex Merchant of Venice by Herobrine
+- HexaRealm Chateau by Firelord
 - Various units by Dom Pedro III
-- HexaRalm Chateau by Firelord
 
 ### Icons
 
-- Worldwide by iconsdb.com for Exploration policy branch
-- Feather by iconsdb.com for Aesthetics policy branch
-- Headdress by Arturo Alejandro Romo Escartin from NounProject.com
-- Track by andriwidodo from NounProject.com
-- Tomahawk by Bakunetsu Kaito from NounProject.com
-- Diamond by SHAHAREA from NounProject.com
-- Ship by Robert Bjurshagen from NounProject.com
+- Anchor by Alvaro Cabrera from NounProject.com
+- Art by scribble.glyph from NounProject.com
+- Artist by my name is mud from NounProject.com
+- Axe by Vectors Market from NounProject.com
```

Comment:
Looks like the Axe was updated, but it's still crediting the old one. Mind making sure the credits are referenced, and that it links to the new axe icon? Want to make sure the artists are credited.